libs/ui/lib/checkbox/Checkbox.tsx

```diff
@@ -43,15 +43,15 @@ export const Checkbox = ({
   className,
   ...inputProps
 }: CheckboxProps) => (
-  <label className="inline-flex items-center">
+  <label className="items-top inline-flex">
     <span className="relative h-4 w-4">
       <input
         className={cn(inputStyle, className)}
         type="checkbox"
         ref={(el) => el && (el.indeterminate = !!indeterminate)}
         {...inputProps}
       />
-      {inputProps.checked && !indeterminate && <Check />}
+      {!indeterminate && <Check />}
```

Comment:
Not sure any of the changes in this file are needed anymore after d14332cfdce7852a32e1c0089264618a78699550. This line in particular seems dubious.